lib/common.js

```diff
@@ -68,18 +69,9 @@ exports.hasProperties = function(obj) {
 
 exports.readJSON = function(file) {
   return asp(fs.readFile)(file)
-  .then(function(pjson) {
-    pjson = pjson.toString();
-    // remove any byte order mark
-    if (pjson.startsWith('\uFEFF')) // TODO: https://www.npmjs.com/package/strip-bom
-      pjson = pjson.substr(1);
```

Comment:
In some cases this is fair point, but ofter modules with large feature set have huge number of submodules. Besides this extra dependency is loaded once (and it is tiny), when jspm is installed and not affecting bundled package. But this is just my thoughts.